react/src/components/MyKeypairInfoModal.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ @license
+ Copyright (c) 2015-2024 Lablup Inc. All rights reserved.
+ */
+import { useSuspendedBackendaiClient } from '../hooks';
+import { useCurrentUserInfo } from '../hooks/backendai';
+import { useTanQuery } from '../hooks/reactQueryAlias';
+import BAIModal, { BAIModalProps } from './BAIModal';
+import Flex from './Flex';
+import { MyKeypairInfoModalQuery } from './__generated__/MyKeypairInfoModalQuery.graphql';
+import { Button, Table, Typography, Tag } from 'antd';
+import graphql from 'babel-plugin-relay/macro';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLazyLoadQuery } from 'react-relay';
+
+interface MyKeypairInfoModalProps extends BAIModalProps {
+  onRequestClose: () => void;
+}
+
+const MyKeypairInfoModal: React.FC<MyKeypairInfoModalProps> = ({
+  onRequestClose,
+  ...baiModalProps
+}) => {
+  const { t } = useTranslation();
+  const [userInfo] = useCurrentUserInfo();
+  const baiClient = useSuspendedBackendaiClient();
+  const { data: keypairs } = useTanQuery({
+    queryKey: ['baiClient.keypair.list', baiModalProps.open], // refetch on open state
+    queryFn: () => {
+      return baiModalProps.open
+        ? baiClient.keypair
+            .list(
+              userInfo.email,
+              ['access_key', 'secret_key', 'is_active'],
+              true,
+            )
+            .then((res: any) => res.keypairs)
+        : null;
+    },
+    staleTime: 0,
+  });
+
+  const supportMainAccessKey = baiClient?.supports('main-access-key');
+
+  const { user } = useLazyLoadQuery<MyKeypairInfoModalQuery>(
+    graphql`
+      query MyKeypairInfoModalQuery($email: String) {
+        user(email: $email) {
+          email
+          main_access_key @since(version: "23.09.7")
+        }
+      }
+    `,
+    {
+      email: userInfo.email,
+    },
+  );
+
+  return (
+    <BAIModal
+      {...baiModalProps}
+      title={t('usersettings.MyKeypairInfo')}
+      centered
+      onCancel={onRequestClose}
+      destroyOnClose
+      width={'auto'}
+      footer={[
+        <Button
+          key="keypairInfoClose"
+          onClick={() => {
+            onRequestClose();
+          }}
+        >
+          {t('button.Close')}
+        </Button>,
+      ]}
+    >
+      <Table
+        scroll={{ x: 'max-content' }}
+        rowKey={'access_key'}
+        dataSource={keypairs}
+        columns={[
+          {
+            title: '#',
+            fixed: 'left',
+            render: (id, record, index) => {
+              ++index;
+              return index;
+            },
+            showSorterTooltip: false,
```

Comment:
It's not a big deal, but you can remove this line because it doesn't support sorting this column.
And I prefer to add this option and sorter directions to Table props to reduce the same codes.